app/services/lti_membership_service.rb

```diff
@@ -0,0 +1,66 @@
+class LtiMembershipService
```

Comment:
Another big question I have that would be helpful in proceeding: do you think this be a service, or should we make it a controller concern? 

Right now, aside from `LtiMembershipService`, we have
-  `LTI OmniAuth strategy` - for launch/auth handshake
- `GitHubClassroom::LtiMessageStore`  - for abstracting launch message redis persistence/retrieval 

And, if we do decide to keep `LtiMembershipService` as a service, should I create an umbrella `GitHubClassroom::LTI` umbrella module that would hold LtiMembershipService, LtiMessageStore, and any other Lti services we may decide to create in the future?